README

```diff
@@ -118,7 +118,7 @@ The rest of this README file contains:
 ===========================================================================
 
 The following abbreviated list of release notes applies to this code
-base as of this writing (March 2017):
+base as of this writing (March 2019):
```

Comment:
We should adjust the month marker from `March` to `April` too since the other PR will be merged in April.

Reply:
yes, and I guess also change the year to 2020 if there are modifications in other sections of the list?

Reply:
both month and year are adjusted now

Reply:
Awesome. Thanks.